DataProviderService/DataProviderService.cs

```diff
@@ -142,23 +142,23 @@ public static void syncFromStarMapService(DateTime? lastSync = null, IProgress<s
         {
             if (StarMapService.Instance != null)
             {
-                Logging.Info("Syncing from EDSM");
-
                 try
                 {
                     List<StarMapResponseLogEntry> flightLogs = StarMapService.Instance.getStarMapLog(lastSync);
-                    Dictionary<string, string> comments = StarMapService.Instance.getStarMapComments();
-                    int total = flightLogs.Count;
-                    int i = 0;
-
-                    while (i < total)
+                    if (flightLogs.Count > 0)
                     {
-                        int batchSize = Math.Min(total, StarMapService.syncBatchSize);
-                        List<StarMapResponseLogEntry> flightLogBatch = flightLogs.Skip(i).Take(batchSize).ToList();
-                        string[] batchNames = flightLogBatch.Select(x => x.system).ToArray();
-                        List<StarSystem> batchsystems = StarSystemSqLiteRepository.Instance.GetOrCreateStarSystems(batchNames, false);
-                        syncEdsmLogBatch(batchsystems, flightLogBatch, comments);
-                        i += batchSize;
+                        Logging.Info("Syncing from EDSM");
```

Comment:
Not sure of the merit of this log entry.

Reply:
Swapped to Logging.Debug